doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1734,7 +1734,8 @@ Number notations
 
          Note that only fully-reduced ground terms (terms containing only
          function application, constructors, inductive type families,
-         sorts, and primitive integers) will be considered for printing.
+         sorts, primitive integers, primitive floats, primitive arrays and the type
```

Comment:
```suggestion
         sorts, primitive integers, primitive floats, primitive arrays and type
```

Reply:
I'm talking about the specific constant `Numbers.Cyclic.Int63.Int63.int` and the two others for floats and arrays. Is the `the` not needed/allowed because I talk about three things and not one?